packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -157,8 +181,12 @@ export class AtomicSearchBox implements AtomicSearchBoxOptions {
         onInput={(e) => this.combobox.onInputChange(e)}
         onKeyUp={(e) => this.combobox.onInputKeyup(e)}
         onKeyDown={(e) => this.combobox.onInputKeydown(e)}
+        tabindex="0"
         type="text"
-        placeholder=""
+        aria-autocomplete="list"
+        aria-controls={this.valuesRef?.id}
+        class="mx-2 my-0 input text-base placeholder-gray-400 border-none outline-none flex flex-grow flex-row align-items-center"
+        placeholder="Search for something"
```

Comment:
I do not think a placeholder is necessary? Google's search box does not have one,

Reply:
It can be removed, I included it because it's in the react vapour design currently.